### ISSUE TITLE
Filter out public suffixes from the blocklist

### ISSUE THREAD

**Issue by groovecoder** [groovecoder/focus-addon#3]
Like https://github.com/mozilla-services/shavar-list-creation/issues/26

**Comment by fmarier** [comment]
We're not currently doing this in the lists we ship, so the add-on should NOT do this in the first iteration.

Also, I believe it only matters for the content category, which we shouldn't waste our time testing IMHO.

**Comment by groovecoder** [comment]
Rather than remove the public suffixes from the blocklist, I plan to implement a `getTopLevelDomain` function that uses the public suffix list to use prefix.public-suffix as the top-level domain.

**Comment by fmarier** [comment]
I don't fully understand what you're planning to do here, but the shavar/platform code doesn't do any transformations to the hostnames provided by Disconnect.

I think the Test Pilot experiment should just do the same as the shavar/platform code.

**Comment by groovecoder** [comment]
Good point. I have code ready for shavar to filter the public suffixes, but since it doesn't do that yet, I'll plan to make the test pilot implementation match the shavar implementation. So I'll remove the "blocker" label here.

FWIW, this is probably a good "next iteration" on both the shavar list creation and the blok experiment.
